Client/Client.cs

```diff
@@ -59,4 +71,26 @@ private static string UserInput(int length, string consoleMessage, string consol
 
         return input;
     }
+
+    public static bool postData(User user)
+    {
+        HttpClient webClient = new HttpClient();
+
+        try
+        {
+            var httpContent = new StringContent(JsonConvert.SerializeObject(user), Encoding.UTF8, "application/json");
+            var httpResponse = webClient.PostAsync("http://localhost:5067/user", httpContent).Result;
+
+            Console.WriteLine(httpResponse.ToString());
```

Comment:
No need to show this when it's successfull. You can put in "Added to queue" or something.